libs/utils/executor.py

```diff
@@ -130,19 +130,28 @@ def __init__(self, target_conf=None, tests_conf=None):
     def run(self):
         self._print_section('Executor', 'Experiments execution')
 
+        self.experiments = []
+
         # Run all the configured experiments
-        exp_idx = 1
         for tc in self._tests_conf['confs']:
             # TARGET: configuration
             if not self._target_configure(tc):
                 continue
             for wl_idx in self._tests_conf['wloads']:
                 # TEST: configuration
-                wload = self._wload_init(tc, wl_idx)
+                wload, test_dir = self._wload_init(tc, wl_idx)
                 for itr_idx in range(1, self._iterations + 1):
-                    # WORKLOAD: execution
-                    self._wload_run(exp_idx, tc, wl_idx, wload, itr_idx)
-                    exp_idx += 1
+                    self.experiments.append({
+                        'wload_name' : wl_idx,
+                        'wload'      : wload,
+                        'conf'       : tc,
+                        'iteration'  : itr_idx,
+                        'out_dir'    : os.path.join(test_dir, str(itr_idx))
```

Comment:
Is it possible to give the `out_dir` a more meaningful name?  Before the traces had relevant names (e.g. `result_latest/fork_migration.dat`.  Now the trace for that experiment would be in `results_latest/1/trace.dat` which is far less readable.